lwgenerate/aws/aws.go

```diff
@@ -299,17 +306,15 @@ func createAwsProvider(args *GenerateAwsTfConfigurationArgs) ([]*hclwrite.Block,
 
 func createLaceworkProvider(args *GenerateAwsTfConfigurationArgs) (*hclwrite.Block, error) {
 	if args.LaceworkProfile != "" {
-		return lwgenerate.NewProvider("lacework",
+		return lwgenerate.NewProvider(
+			"lacework",
```

Comment:
just curious to know if any linting tool you use did this? or that was you formatting it 🤔

Reply:
This was done manually. Interestingly the formatter doesn't format it back to the original layout.
Personally, I find it easier to parse in this format with the args stacked. I can change it back if needs be.

Reply:
I am not against style changes but I have seen others do the same but backward, so we are just playing ping-pong until we have a tool we all use that is opinionated and dictates our desired format/style. Leave it, just some food for thought.

Reply:
Interesting when you consider how opinionated Go and Go fmt is, that it accepts both styles as valid.